cli/flox/src/commands/environment.rs

```diff
@@ -1276,9 +1305,13 @@ pub struct Uninstall {
 }
 
 impl Uninstall {
+    #[instrument(name = "uninstall", fields(packages), skip_all)]
     pub async fn handle(self, mut flox: Flox) -> Result<()> {
         subcommand_metric!("uninstall");
 
+        // We don't know the contents of the packages field when the span is created
+        tracing::Span::current().record("packages", self.packages.iter().join(","));
```

Comment:
do we not?
does `instrument` not work with lists as arguments or can process them (join to string)?

Reply:
`Vec<String>` doesn't implement `tracing::Value` so you have to massage the package list manually.

Reply:
huh weird, but i guess that makes sense, they prob just want you to keep things flat

Reply:
maybe add a comment, since the `tracing::Value` thing is not super obvious (nb)

Reply:
Done